fuse.py

```diff
@@ -441,10 +445,14 @@ class as is to Operations, instead of just the fh field.
         argv = (c_char_p * len(args))(*args)
 
         fuse_ops = fuse_operations()
-        for name, prototype in fuse_operations._fields_:
+        for ent in fuse_operations._fields_:
+            name, prototype = ent[:2]
             if prototype != c_voidp and getattr(operations, name, None):
```

Comment:
The reason the logic is funky in this section is that I was trying to minimize the number of changed lines. You'll notice that the line you are commenting on actually is unchanged.

I can clarify this loop.